test/testenv/verificationutils.go

```diff
@@ -569,3 +569,37 @@ func VerifyAppFrameworkDeployment(standalone *enterprisev1.Standalone) {
 	}
 
 }
+
+// VerifyAppInstall verify that app of specific version is installed
+func VerifyAppInstall(deployment *Deployment, testenvInstance *TestEnv, ns string, podName string, apps map[string]string, versionCheck bool, statusCheck string) {
+	for appName, version := range apps {
+		status, versionInstalled, err := GetPodAppStatus(deployment, podName, ns, appName)
+		gomega.Expect(err).Should(gomega.Equal(""))
+		gomega.Expect(status).Should(gomega.Equal(statusCheck))
+		if versionCheck {
+			gomega.Expect(versionInstalled).Should(gomega.Equal(version))
+		}
+	}
+}
+
+// VerifyAppsCopied verify that apps are copied to correct location based on CR spec
+func VerifyAppsCopied(deployment *Deployment, testenvInstance *TestEnv, ns string, podName string, apps []string) {
+	path := "etc/apps"
+	if strings.Contains(podName, "cluster-master") {
+		path = "etc/master-apps/_cluster/default"
+	}
+	if strings.Contains(podName, "-deployer-") {
+		path = "etc/shcluster/apps"
+	}
+	VerifyAppsInFolder(deployment, testenvInstance, ns, podName, apps, path)
+}
+
+// VerifyAppsInFolder verify that apps are present in folder
+func VerifyAppsInFolder(deployment *Deployment, testenvInstance *TestEnv, ns string, podName string, apps []string, path string) {
+	appList, err := GetSubDirsOnPod(deployment, podName, path)
+	gomega.Expect(err).To(gomega.Succeed(), "Apps not found in path", "Pod", podName)
```

Comment:
Nit: Refine the comment.... As it's not necessary that we failed to get apps but maybe failed to list dirs ?

Reply:
This can be something like `Unable to get apps on pod`